### DISCUSSION TITLE
current_user.is_authenticated lost when redirecting in docker

### DISCUSSION THREAD

**Discussion by hdemann** [maxcountryman/flask-login#689] [Q&A]
Hello,
I have an app using flask-login.  Latest versions of everything.
I have a login page, and using the login_user, I am able to start a session for the user.  I am running this on localhost.  python app.py
However, when I run the same code in a Docker container, the session is lost when I redirect to another page after logging in.
So, I log in, redirected to a page that requires login then  Redirected to login page as there is no session.  
current_user.is_authenticated = False after the redirect.  The docker container is running behind gunicorn/gevent if that might be the cause of the issue.  It works perfectly without the docker container/gunicorn.  This is happening in chrome, did not try another browser

**Comment by hdemann** [comment]
Before anyone goes too far on this one, I think that the problem may be the SESSION_COOKIE_SECURE setting as I do not have my SSL set up in the docker container.  I guess there is a hidden setting in flask that sets this value to False when using localhost?

**Reply by BuchZn** [reply]
Did you find any Solution that is Safe for Production ? Because im Running into the Same Problem. With a nearly smiliar Setup.
